lib/src/test/java/com/auth0/jwt/JWTVerifierTest.java

```diff
@@ -557,12 +568,25 @@ public void shouldThrowOnInvalidJWTId() throws Exception {
     public void shouldRemoveClaimWhenPassingNull() throws Exception {
         Algorithm algorithm = mock(Algorithm.class);
         JWTVerifier verifier = JWTVerifier.init(algorithm)
-                .withIssuer("iss")
-                .withIssuer(null)
+                .withSubject("1234567890")
```

Comment:
@lbalmaceda it's not possible to test this with `null` on issuer since I can't pass `null` to Arrays.asList() varargs parameter because it's `@NotNull` resulting in a NullPointerException.

Reply:
Once you fix the comment I left above about the `Arrays.asList` you can rollback this to how it was before.